Library/Homebrew/test/pathname_spec.rb

```diff
@@ -95,27 +95,6 @@
     end
   end
 
-  describe "#atomic_write" do
```

Comment:
Why are these getting deleted?

Reply:
I thought since we are using a wrapper around `ActiveSupport`, we don't have to test the actual functionality ourselves anymore, but [apparently I was wrong](https://github.com/rails/rails/pull/34037).

Reply:
Good call on the PR 👍